modules/ROOT/pages/starting-and-stopping-mule-esb.adoc

```diff
@@ -0,0 +1,201 @@
+----
+$ $MULE_HOME/bin/mule start
+----
+
+[NOTE]
+====
+You can also start Mule without executing the installation step. This starts Mule in the background for the current user session.
+====
+
+This example starts Mule from a Unix console:
+
+[source,console,linenums]
+----
+$ $MULE_HOME/bin/mule start
+MULE_HOME is set to ~/Downloads/mule-enterprise-standalone-4.1.5
+MULE_BASE is set to ~/Downloads/mule-enterprise-standalone-4.1.5
+Starting Mule Enterprise Edition...
+Waiting for Mule Enterprise Edition.................
+running: PID:87318
+----
+
+== Starting Mule from a Script
+To start Mule from a script or from your IDE without using the Java Service Wrapper, you can use the `org.mule.MuleServer` class. This class accepts a couple of parameters.
+
+----
+org.mule.MuleServer -config mule-config.xml
+----
+
+or
+
+----
+org.mule.MuleServer -builder <fully qualified classname> -config appContext.xml
+----
+
+* *-config* specifies one or more configuration files to use. If this argument is omitted, it will look for and use `mule-config.xml` if it exists.
+
+* *-builder* is a fully qualified classname of the configuration builder to use. If this is not set, the default `org.mule.config.builders.AutoConfigurationBuilder` is used, which will try to auto-detect configuration files based on available builders. In the most common scenario, this will resolve to `org.mule.config.spring.SpringXmlConfigurationBuilder`.
+
+The easiest way to set the classpath is to include all JARs in the `./lib/mule` and `./lib/opt` directories of the distribution. You can look at the dependency report for the server and each of the modules to see exactly which JARs are required for a particular module.
+
+
+== Deploying Mule Applications
+
+. Start Mule.
+. After Mule starts, you can deploy your Mule applications by moving your packaged JAR files to the `apps` directory in `$MULE_HOME`.
+
+== Stopping Mule
+
+Use the `stop` command to stop Mule.
+
+This example stops Mule from a Unix console:
+
+[source,console,linenums]
+----
+$ $MULE_HOME/bin/mule stop
+MULE_HOME is set to /Applications/mule-enterprise-standalone-4.1.5
+MULE_BASE is set to /Applications/mule-enterprise-standalone-4.1.5
+Stopping Mule Enterprise Edition...
+Stopped Mule Enterprise Edition.
+----
+
+== Removing the Mule Service or Daemon
+
+Use the `remove` command to remove the service or daemon from your system.
+
+This example removes the Mule daemon from a Unix console:
+
+[source,console,linenums]
+----
+$ $MULE_HOME/bin/mule remove
+MULE_HOME is set to /Applications/mule-enterprise-standalone-4.1.5
+MULE_BASE is set to /Applications/mule-enterprise-standalone-4.1.5
+Detected Mac OSX:
+Mule Enterprise Edition is not running.
+Removing Mule Enterprise Edition daemon...
+----
+
+== Enable Anypoint Monitoring
+
+Optionally, you can install Anypoint Monitoring for cloud-managed supported versions of on-premises runtimes so you can monitor applications running on that server.
+
+
+== See Also
```

Comment:
Since the article shares some info about the JVM, you could add the following link in See Also:
xref:passing-additional-arguments-to-the-jvm-to-control-mule.adoc[Passing Additional Arguments to the JVM to Control Mule]